### ISSUE TITLE
[Octo-Pro] Adding Persistent Volume Sharing

### ISSUE THREAD

**Issue by ariscahyadi** [nus-ncl/OctoBot#16]
We need to add persistent volume sharing for a movable bot, in order to keep the data or log persistent even though they are moving from one worker to another worker node.

**Comment by ariscahyadi** [comment]
### Manual configuration

#### In the controller node

Install the NFS Server and create the shared folder
```
sudo apt-get install nfs-server
sudo /etc/init.d/nfs-kernel-server restart
sudo mkdir -p /mnt/sharedfolder
sudo chown nobody:nogroup /mnt/sharedfolder
sudo chmod 777 /mnt/sharedfolder
sudo su
echo “/mnt/sharedfolder <Subnet/mask>(rw,sync,no_subtree_check)” >> /etc/exports
exit
```
Create the `pv-claim.yaml`
```
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: task-pv-claim
spec:
  storageClassName: manual
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 3Gi
```
Create `pv-volume.yaml`
``` 
apiVersion: v1
kind: PersistentVolume
metadata:
  name: task-pv-volume
  labels:
    type: local
spec:
  storageClassName: manual
  capacity:
    storage: 10Gi
  accessModes:
    - ReadWriteOnce
  hostPath:
    path: "/mnt/sharedfolder_client"
```
Apply the yaml file
```
kubectl apply -f pv-volume.yaml
kubectl apply -f pv-claim.yaml
```

#### Worker Node

Mount the shared folder
```
sudo mkdir /mnt/sharedfolder_client
sudo mount <NFS_Server>:/mnt/sharedfolder /mnt/sharedfolder_client
```

**Comment by ariscahyadi** [comment]
It has been addressed in PR #30
